doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -215,3 +215,26 @@ cmdline=isolcpus=2
 
 ====
 --
+
+`acpi`::
+
+Configures the ACPI driver.
++
+The only currently supported option is
+[option]`platform_profile`, which sets the ACPI
+platform profile sysfs attribute,
+a generic power/performance preference API for other drivers.
+Multiple profiles can be specified, separated by `|`.
+The first available profile is selected.
++
+--
+.Selecting a platform profile
+====
+----
+[acpi]
+platform_profile=balanced|low-power
```

Comment:
See my other comment, but `low-power` sometimes doesn't exist on certain devices, and is instead `quiet`

Reply:
I updated the example in the doc to the one that is now present in the power-saving profile (`low-power|quiet`).